GenderPayGap.WebUI/wwwroot/styles/components/_accessible-autocomplete.scss

```diff
@@ -0,0 +1,52 @@
+@import "../../assets/stylesheets/accessible-autocomplete.min.css";
+
+.autocomplete-wrapper {
+  max-width: 20em;
+  margin-bottom: 4rem;
+}
+
+.autocomplete__input {
+  background-color: govuk-colour("white");
+  z-index: 101 !important;
```

Comment:
Is there a reason this is `101` and `!important`?

Reply:
Sorry, I copied these over from the old typeahead styling to ensure that it looked the same. I should have remembered to check if they were sensible.

Reply:
I have removed this without any change to the view